needle/core/utils/utils.py

```diff
@@ -195,6 +200,13 @@ def wrapper(obj, *args, **kwargs):
                     self.actual_tries += 1
                     exception = e
                     device.printer.error(exception)
+                    if str(e).find('`') > -1:

```

Comment:
An attempt at trying to resolve issues with accents/backquotes. Currently this does not work, and will actually cause #161 to occur more frequently during the re-attempts. Also once it gets into that state it will error when trying to disconnect and clean up the device causing needle to hang. In which you will need to use `kill <needlePID>` to stop it.